Makefile

```diff
@@ -18,7 +18,7 @@ GO_DEPENDENCIES := $(call rwildcard,pkg/,*.go) $(call rwildcard,cmd/j,*.go)
 
 BRANCH     := $(shell git rev-parse --abbrev-ref HEAD 2> /dev/null  || echo 'unknown')
 BUILD_DATE := $(shell date +%Y%m%d-%H:%M:%S)
-CGO_ENABLED = 0
+CGO_ENABLED = 1
```

Comment:
why?

Reply:
No, but there is a dependency that does. Deep down...

Reply:
So compilation fails without it

Reply:
Because of the go-upgrade? I will take a look ... cgo opens up hell on earth :grimacing: